modules/client/examples/05-encoders-decoders/12-token-voting/03-update-plugin-settings.ts

```diff
@@ -36,3 +31,35 @@ const pluginAddress: string = "0x1234567890123456789012345678901234567890"; // t
 // Updates the configuration of a TokenVoting plugin for a DAO.
 const updatePluginSettingsAction: DaoAction = tokenVotingClient.encoding.updatePluginSettingsAction(pluginAddress, configActionPrarms);
 console.log({ updatePluginSettingsAction });
+
+
+/* MARKDOWN
+Returns:
+
+```
+{
+  to: "0x1234567890...",
+  value: 0n,
+  data: Uint8Array[12,34,45...]
+}
+```
+
+### Decoding
+*/
+
+// Decodes the parameters of an update plugin settings action.
+const decodeUpdateTokenVotingSettings: VotingSettings = tokenVotingClient.decoding.updatePluginSettingsAction(data);
+console.log({ decodeUpdateTokenVotingSettings });
+
+/* MARKDOWN
+Returns:
+
+```
+{
+  minDuration: 7200, // seconds
+  minParticipation: 0.25, // 25%
+  supportThreshold: 0.5, // 50%
+  minProposerVotingPower: BigInt("5000")
```

Comment:
`BigInt(...)` is used as a "constructor" for legacy environments, but the value you are going to receive would be expressed like `5000n` by a browser

Reply:
Yes, the comments also won't be returned and so on. I just copied it over. Someone should go over all the example and polish this